src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -79,6 +79,10 @@ recompute (no ¬a) a = ⊥-elim (¬a a)
 infixr 1 _⊎-dec_
 infixr 2 _×-dec_ _→-dec_
 
+T? : ∀ x → Dec (T x)
+does  (T? x) = x
+proof (T? x) = T-reflects x
+
```

Comment:
I think this is more pungently expressed in terms of `_because_`?
```agda
T? x = x because (T-reflects x)
```

Reply:
Sure!